stankove_napady.md

```diff
@@ -0,0 +1,105 @@
+Zkusim popsat svuj navrh a zkusme to spolecnymi pripominkami upravit do lepsi podoby. Je to moje stankova premiera.
+<br>
+</br>
+## Co chceme prezentovat?
+
+**python.cz** 
+
+> aby je navstevovalo vice pythonistu i zacatecniku zajemcu o python
+
+> aby vedeli jake informace tam najdou
+
+**pyvo.cz**
+
+> protoze lide maji vedet, ze kazda zasadni zmena okolo mista ci casu srazu by se mela jako prvni objevit predevsim tam? tzn. naucit lidi hledat ze vsech kanalu dulezito info prave tady
+
+**PyLadies a Advanced srazy**
+
+> zvysit povedomi o PyLadies celkove (stale se setkavam jako organizatorka s tim, ze spousty python lidi o teto aktivite lidi z komunity vubec nevi)
+
+> zvysit povedomi i u mensich firem, ktere by to jakkoliv rady podporily
+
+> moznost jak zapojit zase nove kouce, kteri by napriklad opravdu radi Python ucili o moznosti na PyLadies nevi
+
+**naucse.python.cz**
+
+> otazka na @encukou  zdali jsme schopni za mesic posunou tuto novou vec tak, abychom ji tam mohli prezentovat?
+
+> vidim to jako velmi zajimave doplneni stanku vedle PyLadies - bude pekne videt, ze mame zajem ucit vsechny bez rozdilu
+
+**Zapojse a 📚  blog.python.cz**
+
+> to by tam nemelo chybet
+
+**PyCon?**
+> ze bude?
+
+<br>
+</br>
+## Kdo to muze prezentovat - sonda do vaseho casoveho rozvrhu
```

Comment:
My se @zuzejk nemame jeste program na ten vikend, ale nevim, jestli se nam tam jeste neco nevyvrbi. Ja furt uvazuju, ze bych ten dalsi vikend jel na PyCon PL a to bych asi nechtel pak mit dve takovy akce za sebou. Na prednasky vubec nemusim, z toho co jsem se dival jde vetsina uplne mimo me.